src/Domain/DomainHelpers.cpp

```diff
@@ -349,8 +349,9 @@ void set_cartesian_periodic_boundaries(
     const gsl::not_null<
         std::vector<DirectionMap<VolumeDim, BlockNeighbor<VolumeDim>>>*>
         neighbors_of_all_blocks) {
-  ASSERT(orientations_of_all_blocks.size() == corners_of_all_blocks.size(),
-         "Each block must have an OrientationMap relative to an edifice.");
+  if (orientations_of_all_blocks.size() != corners_of_all_blocks.size()) {
+    ERROR("Each block must have an OrientationMap relative to an edifice.");
```

Comment:
what is an edifice?

Reply:
I have no idea, sorry. You'll have to ask the original author, which looks like is Marcie based on the git log.